### DISCUSSION TITLE
How do I deploy it to vercel?

### DISCUSSION THREAD

**Discussion by billowssun** [open-webui/open-webui#4023] [General]
![image](https://github.com/user-attachments/assets/e54b6288-2930-4b70-a14c-1eff8bab6603)
![image](https://github.com/user-attachments/assets/b9f1bc04-e9f0-466c-88f5-e6333c502dd4)

**Comment by solomonshalom** [comment]
Same Q here! I am not so sure on hosting this on a personal machine but would love to do it through Vercel!

**Reply by billowssun** [reply]
> I'm having the same problem! I'm not quite sure if I want to host it on my PC, but would love to host it through Vercel!

I purchased a server for deploying openwebui, bro, how did you fix it?

**Reply by solomonshalom** [reply]
I think I am just going to stick with Render 😭

**Reply by Shey3ge636** [reply]
Did you managed to do it on vercel ??

**Reply by solomonshalom** [reply]
No :(

**Reply by salictou** [reply]
@solomonshalom have you managed to successfully deploy it on render? If so could you kindly share the render.yaml file?

**Reply by Yashh56** [reply]
Hey, do you have any updates on the deployment on Vercel or render?

**Comment by la3rence** [comment]
I deployed it on HuggingFace Space: https://huggingface.co/spaces/lawrenceli/open-ui
It requires some ENV variables in .env.example. You just need set the DATABASE_URL and WEBUI_SECRET_KEY.

**Reply by billowssun** [reply]
Ok,thanks you

**Reply by Yashh56** [reply]
> I deployed it on HuggingFace Space: https://huggingface.co/spaces/lawrenceli/open-ui It requires some ENV variables in .env.example. You just need set the DATABASE_URL and WEBUI_SECRET_KEY.

which database URL should be required?
